### PR TITLE
docs: create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by Anirudh3167**
# Motive
The PR template created

# Issue this PR is related
Reffers to Issue: #12

https://github.com/Showrin/nextjs-tailwind-storybook-with-ci-cd/issues/12

# Additional Description
comments are added in the `PULL_REQUEST_TEMPLATE.md` to clear the intention of adding each and every heading

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Description
+<!-- Detailing of the problem that this PR is solving -->
+
+# Issue
+<!-- A PR can either be an issue or it is independently made changes -->
```
Comment (Showrin) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
# Issue
N/A <!-- A PR can either be an issue or it independently made changes. If it's created to resolve an issue, please provide the issue number in this format - (#issue_number) -->

# Description
<!-- Detailing of the problem that this PR is solving -->
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<!-- Specially for those cases where there satisfying one/few of the tasks mentioned in the issue -->
+<!-- Or might be some additional changes that are not covered by the Issue -->
+
+# Checklist
+- [ ] The code follows the project's style guidelines and conventions.
+- [ ] All existing tests pass successfully.
+- [ ] New tests have been added to cover the changes or improvements.
+- [ ] Documentation has been updated or created to reflect the changes made.
+- [ ] The commit messages are clear and descriptive.
+- [ ] The pull request title provides a concise summary of the changes.
```
Comment (Showrin) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] The pull request title concisely summarizes the changes.
```

**Review by Showrin — CHANGES_REQUESTED** [review]
Please, write the commit messages following this **[commit convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)**. Thanks for the contribution.

**Comment by Anirudh3167** [comment]
I have edited the file as you suggested

This time I have also added a screenshots section

so as to make it easy for the reviewer to know about the frontend changes

**Comment by Showrin** [comment]
> I have edited the file as you suggested
> 
> This time I have also added a screenshots section
> 
> so as to make it easy for the reviewer to know about the frontend changes

Thanks, @Anirudh3167 for the contribution.

**Review by Showrin — APPROVED** [review]

**Pull request merged**